data/src/test/java/com/android254/data/dao/SessionDaoTest.kt

```diff
@@ -15,51 +15,44 @@
  */
 package com.android254.data.dao
 
-import android.content.Context
-import androidx.room.Room
-import androidx.test.core.app.ApplicationProvider
-import com.android254.data.db.Database
 import com.android254.data.db.model.Session
+import com.google.common.truth.Truth
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
 import kotlinx.coroutines.test.runTest
 import kotlinx.datetime.toInstant
-import org.hamcrest.CoreMatchers.`is`
-import org.hamcrest.MatcherAssert.assertThat
-import org.junit.After
 import org.junit.Before
 import org.junit.Test
-import org.junit.runner.RunWith
-import org.robolectric.RobolectricTestRunner
-import java.io.IOException
 
-@RunWith(RobolectricTestRunner::class)
 class SessionDaoTest {
-
+    private lateinit var session: Session
+    @MockK
     private lateinit var sessionDao: SessionDao
-    private lateinit var db: Database
```

Comment:
We are using Robolectric because we want to test against a real db.

Reply:
@michaelbukachi With robolectric: it slows down the tests a lot, but it's sometimes helpful if we really want to test the integration with the Android system

Reply:
I'd suggest to use it only in a few places, not everywhere. Considering that tests performance are also very important and  should be fast as possible

Reply:
> We are using Robolectric because we want to test against a real db.

But it's okay if you guys think you want to stick with this.

Reply:
@anselmoalexandre We are aware of its drawbacks. We used it to ensure all our queries were running as expected.